app/stock-landing/src/page/home/index.style.ts

```diff
@@ -0,0 +1,48 @@
+import styled from '@emotion/styled';
+
+const Container = styled.div`
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 0rem 1.5rem;
+  text-align: center;
+`;
+
+const Title = styled.h1`
+  font-size: 3rem;
+  margin-bottom: 0.5rem;
+`;
+
+const Subtitle = styled.p`
+  margin-bottom: 1.5rem;
+  font-size: 1.2rem;
+`;
+
+const Button = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+  padding: 12px 24px;
+  margin-top: 3rem;
+  background: rgba(0, 103, 224, 1);
```

Comment:
```suggestion
  background: #5865F2;
```

https://discord.com/branding
디스코드 브랜딩 색에 맞추면 좋을듯해요!